src/Localization/Localization/src/ResourceManagerStringLocalizer.cs

```diff
@@ -221,5 +237,11 @@ private static partial class Log
     {
         [LoggerMessage(1, LogLevel.Debug, $"{nameof(ResourceManagerStringLocalizer)} searched for '{{Key}}' in '{{LocationSearched}}' with culture '{{Culture}}'.", EventName = "SearchedLocation")]
         public static partial void SearchedLocation(ILogger logger, string key, string locationSearched, CultureInfo culture);
+
+        [LoggerMessage(2, LogLevel.Debug, $"The resource with key '{{Key}}' is not found.", EventName = "ResourceNotFound")]
+        public static partial void ResourceNotFound(ILogger logger, string key);
+
+        [LoggerMessage(3, LogLevel.Debug, $"The resource with key '{{Key}}' is found.", EventName = "ResourceFound")]
```

Comment:
```suggestion
        [LoggerMessage(3, LogLevel.Debug, $"A resource for key '{{Key}}' with culture '{{Culture}}' was found.", EventName = "ResourceFound")]
```

Reply:
So, you will the non found resources?

Reply:
I'm not sure I understand your question, but I'm asking that we only add the "ResourceNotFound" log not the "ResourceFound" log.

Reply:
Forgot my above comment seems I miss some words during the writing :)